app/javascript/tests/subject/monitoring_actions/MonitoringStatus.test.js

```diff
@@ -13,24 +13,31 @@ const authyToken = 'Q1z4yZXLdN+tZod6dBSIlMbZ3yWAUFdY44U06QWffEP76nx1WGMHIz8rYxEU
 const monitoringStatusValues = [ 'Actively Monitoring', 'Not Monitoring' ];
 const monitoringReasons = [
```

Comment:
nitpick and I totally did this originally but can you just change this to monitoringReasonValues to match the other one